src/gradualizer_db.erl

```diff
@@ -491,12 +501,12 @@ collect_specs(Module, Forms) ->
 
 normalize_spec({{Func, Arity}, Types}, Module) ->
     {{Module, Func, Arity}, Types};
-normalize_spec(Spec = {{M, F, A}, _Types}, Module) ->
-    case M of
-        Module -> ok;
-        _ -> error_logger:info_report([{spec_for, {M,F,A}},
-                                       {found_in, Module}])
-    end,
+normalize_spec(Spec = {{_M, _F, _A}, _Types}, _Module) ->
+    %case M of
+    %    Module -> ok;
+    %    _ -> error_logger:info_report([{spec_for, {M,F,A}},
+    %                                   {found_in, Module}])
+    %end,
```

Comment:
I suggest removing this code instead of keeping it in comments.

Reply:
Done